conf/local.conf.sample

```diff
@@ -59,7 +59,7 @@ ENABLE_UART = "1"
 INITRAMFS_IMAGE = "ww-console-image-initramfs"
 # If we want to bundle initramfs, we need to fix initramfs so that /dev/console exists.
 # Otherwise the board will not boot.
-INITRAMFS_IMAGE_BUNDLE = "1"
+#INITRAMFS_IMAGE_BUNDLE = "1"
```

Comment:
probably just delete instead of comment?

Reply:
yeah, you're right. this option has been enabled, then disabled, and reenabled a few times now.